docs/pages/foundations/messaging/components.js

```diff
@@ -0,0 +1,222 @@
+import MainSection from '../../../docs-components/MainSection.js';
+import Page from '../../../docs-components/Page.js';
+import PageHeader from '../../../docs-components/PageHeader.js';
+
+export default function MessagingComponentsPage(): Node {
+  return (
+    <Page title="Messaging components">
+      <PageHeader name="Messaging components" type="guidelines" />
+      <MainSection name="Callout">
+        <Example
+          id="calloutExample"
+          name="Callout example"
+          showCode={false}
+          showHeading={false}
+          defaultCode={`<Callout
+  title='Your tag is not working'
+  message='Data may be outdated because your tag is not sendng information. Fix your tag for the most accurate metrics.'
+  type='error'
+  dismissButton={{
+    accessibilityLabel: 'Dismiss banner',
+    onDismiss: () => {},
+  }}
+  iconAccessibilityLabel="Error"
+  primaryAction={{
+    accessibilityLabel: 'Fix tag',
+    label: 'Fix tag',
+    onClick: () => {},
+  }}
+/>`}
+        />
+        <Flex direction="column" gap={4}>
+          <Text>
+            Banners sit on a page or surface and provide status alerts, information or
+            recommendations to a user. They are used for highest priority messages, usually at the
+            top of a page.
+          </Text>
+
+          <Text>
+            <Link href="https://gestalt.pinterest.systems/web/callout" underline="always">
+              Go to the Callout component
+            </Link>
+          </Text>
+        </Flex>
+      </MainSection>
+      <MainSection name="SlimBanner">
+        <Example
+          id="slimbannerExample"
+          name="SlimBanner example"
+          showCode={false}
+          showHeading={false}
+          defaultCode={`<SlimBanner
+  dismissButton={{
+    accessibilityLabel: 'Dismiss banner',
+    onDismiss: () => {},
+  }}
+  iconAccessibilityLabel="Information"
+  message="Idea Pins are now available across platforms."
+  primaryAction={{
+    accessibilityLabel: 'Apply for access',
+    label: 'Apply for access',
+    onClick: () => {},
+  }}
+  type="info"
+/>`}
+        />
+        <Flex direction="column" gap={4}>
+          <Text>
+            SlimBanner conveys brief information related to a specific section of a page. The
+            message can relay success, warning, error or general information. SlimBanners are used
+            to reference a specific section of a page, or in any dense interface where space is a
+            concern.
+          </Text>
+
+          <Text>
+            <Link href="https://gestalt.pinterest.systems/web/slimbanner" underline="always">
+              Go to the SlimBanner componen
+            </Link>
+          </Text>
+        </Flex>
+      </MainSection>
+      <MainSection name="Modal">
+        <SandpackExample code={modalExample} name="Modal Main Example" hideEditor />
+        <Flex direction="column" gap={4}>
+          <Text>
```

Comment:
I would change to 

 "A modal dialog can be used to alert a user of an issue, or to request confirmation after
            a user-generated action. **Modal overlays and blocks page** content until it is
            dismissed by the user."

until we change the name

Reply:
Done!